### ISSUE TITLE
Provide HTML-Rendered Dashboard

### ISSUE THREAD

**Issue by mmccool** [eclipse-thingweb/domus-tdd-api#17]
It would be helpful to provide an entry point that provides HTML that allows the content of the directory to be navigated by a human using a web browser, in particular to grab URLs to TDs and perhaps also to upload TDs (and also delete them, see their validation status, etc.).

**Comment by wiresio** [comment]
Like [this](http://plugfest.thingweb.io:8081/things?format=treeview)?

**Comment by danielpeintner** [comment]
The tree view is (I think) an improvement, but it gives the impression one can interact with the tree (folding in and out) but that is not the case...

**Comment by wiresio** [comment]
> EDIT: I noticed click work (later) on nested levels but _not_ on the root level!?

Yes, the behaviour is a bit strange. Sometimes you have to "double click", root cannot be collapsed, etc. - that's why I suppose to implement an external application coded by an external person ;-)
